connectors/src/api/webhooks/webhook_google_drive.ts

```diff
@@ -39,7 +41,15 @@ const _webhookGoogleDriveAPIHandler = async (
   const workflowRes = await launchGoogleDriveIncrementalSyncWorkflow(
     webhook.connectorId.toString()
   );
+
   if (workflowRes.isErr()) {
+    if (workflowRes.error instanceof RateLimitError) {
+      logger.info(
+        {},
```

Comment:
Let’s add a log when we do signal ?

Reply:
To convince ourselves that everything works as expected?

Reply:
The case where we send the signal does not change. The workflow starts, there is already a log for that and we see the signal on the Temporal side so I think it's enough.